src/GUI/EFCorePowerTools/Extensions/EnvDTEExtensions.cs

```diff
@@ -84,9 +86,21 @@ private static Project GetProject(DTE dte, string projectItemPath)
             var projects = Projects(dte);
             foreach (var project in projects)
             {
-                if (project.FullName == projectItemPath)
```

Comment:
Is this something you have faced in real life?

Reply:
Indeed, while debugging earlier today.
It would randomly fail and threw NotImplementedExceptions when accessing the `FullName` property, both from the compiled code and the IntelliSense.
Therefore the fallback using the unique name.